ovmf-vars-generator

```diff
@@ -48,6 +48,7 @@ def generate_qemu_cmd(args, readonly, *extra_args):
         '-display', 'none',
         '-no-user-config',
         '-nodefaults',
+        '-no-reboot',
```

Comment:
Ah-ha, makes sense; looks good to me.

Reviewed-by: Kashyap Chamarthy <kchamart@redhat.com>